src/html/index.ejs

```diff
@@ -39,10 +39,9 @@
 					<form class="navbar-form navbar-right">
 						<div class="form-group">
 							<select style="height:34px;padding-right: 5px;" id="exampleTasks" class="taskselect" selected="false" onchange="selectExampleTask()">>
-	 					  	    <option disabled selected value> -- select an example task -- </option>
-  								<option value="binary;capybaras.osexp">Capybars</option>
-  								<option value="binary;GazecuingTheeyeshaveitAndroid.osexp">Gaze Cuing</option>
-  								<option value="binary;stroop.osexp">Stroop</option>
+	 					  	    <option disabled selected value> Example experiments … </option>
+  								<option value="text;lexical-decision.osexp">Lexical decision</option>
+  								<option value="binary;gaze-cuing.osexp">Gaze cuing</option>
```

Comment:
Where do you process this binary vs text prefix? Or was this feature added during one of the previous commits? It's new to me at least, but if it already works...